### PR TITLE
Reorder the use Analytics snippets info.

### PR DESCRIPTION
**Pull request by asvinb**
## Summary

Addresses issue #3996 

## Checklist

- [x] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: assets/js/modules/analytics/components/settings/SettingsView.js [hunk 1]
```diff
@@ -77,6 +77,9 @@ export default function SettingsView() {
 	const canUseSnippet = useSelect( ( select ) =>
 		select( MODULES_ANALYTICS ).getCanUseSnippet()
 	);
+	const useGA4Snippet = useSelect( ( select ) =>
+		select( MODULES_ANALYTICS_4 ).getUseSnippet()
```
Comment (eugene-manuilov) [inline assets/js/modules/analytics/components/settings/SettingsView.js:81]: We still use the `ga4setup` feature flag in this component, so let's call the `getUseSnippet` selector only if the `isGA4Enabled` is truthy.

**Review by eugene-manuilov — CHANGES_REQUESTED** [review]
Thanks, @asvinb. Two minor comments and this should be good to go. :+1:

File: assets/js/modules/analytics/components/settings/SettingsView.js (hunk 1 above)
Comment (asvinb) [reply assets/js/modules/analytics/components/settings/SettingsView.js]: 👍

File: assets/js/modules/analytics/components/settings/SettingsView.stories.js [hunk 2]
```diff
@@ -33,28 +33,41 @@ import * as ga4Fixtures from '../../../analytics-4/datastore/__fixtures__';
 
 const features = [ 'ga4setup' ];
 
-function Template() {
+const Template = ( { setupRegistry = () => {}, ...args } ) => {
```
[the remark this answers is not in the document]
Comment (asvinb) [reply assets/js/modules/analytics/components/settings/SettingsView.stories.js]: Sounds good. Updated!

**Review by eugene-manuilov — APPROVED** [review]
LGTM. Thanks, @asvinb.

**Pull request merged**